packages/core/src/tracing/index.js

```diff
@@ -179,6 +180,13 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL === 'debug') {
+    const method = getPackageInstallation();
+
+    // eslint-disable-next-line no-console
+    console.debug(`The App has instrumented instana using: ${method}`);
```

Comment:
issue: I am getting

> The App has instrumented instana using: no additional flags

When doing

> $ INSTANA_DEBUG=true node --require ~/dev/instana/nodejs-2/packages/collector/src/immediate.js app.js

Reply:
I am getting correctly!! but why🤔 
<img width="1313" alt="Screenshot 2024-08-22 at 3 30 33 PM" src="https://github.com/user-attachments/assets/233f0bb9-8235-43cd-9050-30a50f8f309b">

Reply:
The path is different. Maybe we only cover specific use cases?

Reply:
The reason is that you are referring to the packages directly. 
The logic behind the comparison is checking for the `@instana` keyword in the arguments (as the clients are using our npm package).
So in a client app perspective it should work if they are including our package in the application start arguments

Reply:
We could simplify the whole function by printing all values if `--require`, `--import` or `--experimental-loader` is used.

e.g. customer uses `node --import X --import X app.js`

**Advantages:**
- less complicated logic, less checks
- covers the use case that if a customer uses multiple import values to dig down potential problems -> conflicts
- this is only debug information
- covers more use cases (such as the case I showed)

The output would be:

```
console.debug(`The App is using the following preload flags: ${preloadFlags}`);
```

Reply:
Yeah perfect 👍